README.md

```diff
@@ -168,6 +168,12 @@ Note: Don't use `@` symbol when adding username.
 When you are done with the setup, you should host your website online.
 We highly recommend to read through the [Deploying on Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) docs for React.
 
+#### Configuring GitHub Actions
+
+- [Create a second GitHub Personal Access Token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) for use by the [JamesIves GitHub Pages Deploy GitHub Actions](https://github.com/JamesIves/github-pages-deploy-action) with `public_repo, read:org, workflow` checked
```

Comment:
Change this line to,
Create a [GitHub CLI Personal Access Token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) with access to `public_repo, read:org, workflow`, to setup a [GitHub action](https://github.com/JamesIves/github-pages-deploy-action) to deploy to GitHub pages.

Reply:
Note: A second token is not required for GitHub action  as itself provides one. Use  `${{ secrets.GITHUB_TOKEN }}` to get it.

Reply:
Remove the above line.

Reply:
@naveen521kk committed the changes asked.  Just wanted to point out that https://github.com/JamesIves/github-pages-deploy-action#required-setup talks about how `GITHUB_TOKEN` will not always work depending on repository permissions.  I have my fork of this repository set to public and I couldn't quite find the specifics needed for `GITHUB_TOKEN` to work.  I haven't tried it again recently but last week this was failing with commit permissions missing when the action tried to commit back to the `gh-pages` branch. 

Hence, why I used a second personal access token to supply as the `ACCESS_TOKEN` repository secret